gpii/configs/fm.ps.sr.dr.mm.os.lms.multipleMatchMakers.json

```diff
@@ -1,7 +1,7 @@
 {
```

Comment:
This file should just be deleted, imo. The file `../node_modules/matchMaker/configs/proxy.json` is dead and gone, and for the immediate future, we will only have one MM per system configuration